api/models/Notification.js

```diff
@@ -134,46 +134,50 @@ module.exports = bookshelf.Model.extend({
   sendEventInvitationPush: function () {
     const post = this.post()
     const actor = this.actor()
+    const locale = this.reader().get('settings')?.locale || this.actor().get('settings')?.locale || 'en'
```

Comment:
I think it should always be the reader, not the actor right? though I guess that could be a way to guess if we dont have the reader's locale

Reply:
Discussion for this here: https://www.hylo.com/my/posts/post/60053

Reply:
ah hmm, ok this is fine for now I guess 🤷‍♂️

Reply:
Do we want give all users a default locale?